### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by lauranolling**
Adapted from the [Energy Track and Trace](https://github.com/Energy-Track-and-Trace) project.

### CODE REVIEW

**Comment by lauranolling** [comment]
This template is not done. I need to make a document describing the use of RFCs in this project, and then insert a link into this template, for more guidance to contributors who may need it.

**Comment by lenucksi** [comment]
Added @wisbech as he might be able to provide some practical input and review based on the experiences within the https://github.com/Energy-Track-and-Trace project.

File: .github/ISSUE_TEMPLATE/rfc-template.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+name: RFC Template
+about: Fill in this template and you are on your way to submit an RFC to the Project-Origin
+  working group.
+title: "[RFC]000 - Issue title"
+labels: rfc
+assignees: ''
+
+---
+
+---
```
Comment (lenucksi) [inline .github/ISSUE_TEMPLATE/rfc-template.md:11]: This looks sort of duplicated. Did you verify both metadata sections are required?

File: .github/ISSUE_TEMPLATE/rfc-template.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+about: Fill in this template and you are on your way to submit an RFC to the Project Origin working group.
+title: "[RFC]000 - Issue title"
+labels: RFC
+assignees: ''
+---
+# 000-Template
+
+### Note: This template for creating RFCs has been adapted from the template used in the [Energy Track and Trace](https://github.com/Energy-Track-and-Trace) project.
+
+Learn more about [using RFCs in this project](missing link).
```
Comment (lenucksi) [inline .github/ISSUE_TEMPLATE/rfc-template.md:22]: The original project had a pointer to this page I think.
```suggestion
Learn more about [using RFCs in this project](https://github.com/Energy-Track-and-Trace/ett-documentation/tree/main/rfcs).
```
Might not be perfect for us, but it's still a good page so I added it. Maybe we can adapt it for our needs?
Also introducing the way to use this for our local project will need some interactive work I think :)

**Review by lenucksi — APPROVED** [review]
We know this works elsewhere, so let's see what adaptations for our local project we need. Two small suggestions, rest looks fine :)

File: .github/ISSUE_TEMPLATE/rfc-template.md (hunk 2 above)
Comment (wisbech) [reply .github/ISSUE_TEMPLATE/rfc-template.md:22]: @lenucksi you are absolutely right - some interactive work would be a welcome addition to the template and would ease adoption. 
In addition we might have to consider word templates, hence word is the modus operandi for most non-technical potential commiters.

File: .github/ISSUE_TEMPLATE/rfc-template.md (hunk 1 above)
Comment (wisbech) [reply .github/ISSUE_TEMPLATE/rfc-template.md:11]: I believe these metadata are for [Issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) on github - that is why they are duplicated, consider them a markdown header :-)

**Review by wisbech — APPROVED** [review]
Looking good to me - for now.

Congratulations @lauranolling btw on your first PR 👍

**Comment by lenucksi** [comment]
Time for @lauranolling to click the big green "Squash and merge" button then and get it merged into the repo!

**Pull request merged**